docs/manual/build.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 mkdir -p _shake
-ghc --make Build.hs -rtsopts -with-rtsopts=-I0 -outputdir=_shake -o _shake/build && _shake/build "$@"
+command -v stack >/dev/null 2>&1
+if [ $? -eq 0 ]; then
```

Comment:
I tend to use `if type stack > /dev/null 2>&1; then` as the guard here. Any reason to prefer your formulation or mine? (I'm not a big shell scripter, so defer to your knowledge here!)